cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/Embankment.java

```diff
@@ -42,7 +44,10 @@
 @JsonDeserialize(builder = Embankment.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+@JsonPropertyOrder({"projectIdentifier", "location"})
```

Comment:
Is it possible to set the order for "this"?

Reply:
Meaning an annotation that adjusts the property order for Embankment DTO? I think your choices are code order, explicitly specified order or alphabetic
Or you want a way to adjust the order on an individual property? I think you can set an index # inside @JsonProperty annotations.  Not sure how it deals with ties.

Or what do you mean by 'this'?

Reply:
The attribute data that the embankment itself supplies. For example:
upstreamSideSlope
downstreamSideSlope
structureLength
Although from the javadoc, it implies that projectIdentifier is first, then location, then the "this" or remainder of embankment attributes.
https://fasterxml.github.io/jackson-annotations/javadoc/2.8/com/fasterxml/jackson/annotation/JsonPropertyOrder.html
I asked this question because in https://github.com/USACE/cwms-data-api/blob/14c96c84812ab340f3dda8897fa15282199fcb7a/cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/embankment.json the order is 
project identifier,
"this",
location.